corpus/transform.py

```diff
@@ -345,6 +346,11 @@ def run(self):
                 for rec in c.all_recordings():
                     merged_corpus.add_recording(rec)
                 merged_corpus.speakers.update(c.speakers)
+            elif self.merge_strategy == MergeStrategy.CONCATENATE:
+                for subcorpus in c.top_level_subcorpora():
+                    merged_corpus.add_subcorpus(subcorpus)
+                for rec in c.top_level_recordings():
+                    merged_corpus.add_recording(rec)
```

Comment:
Do we also have to add the top level speakers?
```suggestion
                    merged_corpus.add_recording(rec)
                for speaker in c.top_level_speakers():
                    merged_corpus.add_speaker(speaker)
```

(with a suitable implementation of `Corpus.top_level_speakers()`)